pkgs/development/libraries/gfbgraph/default.nix

```diff
@@ -1,23 +1,35 @@
 { stdenv, fetchurl, pkgconfig, glib, librest, gnome-online-accounts
-, gnome3, libsoup, json-glib, gobject-introspection }:
+, gnome3, libsoup, json-glib, gobject-introspection
+, gtk-doc, pkgs, autoreconfHook }:
 
 stdenv.mkDerivation rec {
   pname = "gfbgraph";
-  version = "0.2.3";
+  version = "0.2.4";
 
-  outputs = [ "out" "dev" "devdoc" ];
+  outputs = [ "out" "dev" ];
 
   src = fetchurl {
     url = "mirror://gnome/sources/${pname}/${stdenv.lib.versions.majorMinor version}/${pname}-${version}.tar.xz";
-    sha256 = "1dp0v8ia35fxs9yhnqpxj3ir5lh018jlbiwifjfn8ayy7h47j4fs";
+    sha256 = "0yck7dwvjk16a52nafjpi0a39rxwmg0w833brj45acz76lgkjrb0";
   };
 
-  nativeBuildInputs = [ pkgconfig gobject-introspection ];
+  nativeBuildInputs = [ pkgconfig gobject-introspection gtk-doc autoreconfHook ];
   buildInputs = [ glib gnome-online-accounts ];
   propagatedBuildInputs = [ libsoup json-glib librest ];
 
   configureFlags = [ "--enable-introspection" ];
 
+  patchPhase = ''
+    substituteInPlace autogen.sh \
+      --replace "#!/bin/sh" "#!${pkgs.bash}/bin/bash" \
+      --replace "which" "${pkgs.which}/bin/which"
+  '';
+
+  autoreconfPhase = ''
```

Comment:
At this point we are not using `autoreconfHook` so we should just run `NOCONFIGURE=1 ./autogen.sh` in `preConfigure` and add `autoconf automake libtool` to `nativeBuildInputs`.

Reply:
This does not seem resolved.

Reply:
Sorry totally overseen that. Now it should be covered.